pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[tool.pyright]
+
+include = [
+    "shimmy/**",
+    "test/**"
+]
+
+exclude = [
+    "**/node_modules",
+    "**/__pycache__",
+]
+
+strict = [
+
+]
+
+typeCheckingMode = "basic"
+pythonVersion = "3.6"
+pythonPlatform = "All"
+enableTypeIgnoreComments = true
+
+# This is required as the CI pre-commit does not download the module (i.e. numpy, pygame, box2d)
+# Therefore, we have to ignore missing imports
+reportMissingImports = "none"
+# Some modules are missing type stubs, which is an issue when running pyright locally
+reportMissingTypeStubs = false
+# For warning and error, will raise an error when
+reportInvalidTypeVarUse = "none"
+
+reportUnknownMemberType = "warning"
```

Comment:
Remove all of these options, this was only as gym has terrible typing and as going to be more of issue to fix them so this was to remove all of the errors